packages/template/package-template.json

```
@@ -98,13 +98,13 @@
   },
   "//": "IF_PLATFORM react-like",
   "peerDependencies": {
-    "@types/react": ">=18.2 || >=19.0.0-rc.0",
+    "@types/react": ">=19.0.0-rc.0",
     "//": "IF_PLATFORM next",
```

Biome (2.1.2)

[error] The key // was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)